### DISCUSSION TITLE
Adding upcoming books

### DISCUSSION THREAD

**Discussion by PhilippKammann** [gotson/komga#1633] [Ideas]
My feature idea would be to have the ability to add “pseudo”-books that will release in the future.

Similar to unavailable books, they wouldn't have a file and only exist in the Komga DB, but also not be removed by emptying the trash.

**Purpose:**
I follow a lot of series, and it's hard to keep track of them all. So, I would regularly check publisher sites to see if there was a new release or if there is one in the coming months.
The upcoming books, I would like to add to Komga, so I can keep track of them, easily notice if a new book was released, and give my users an overview of books that I'm missing.

**Example:**
- Jellyfin
![6K0rX99](https://github.com/user-attachments/assets/78d9c0d0-b1a7-4f6a-a2f7-8d77dd7ee8a1)

**Comment by gotson** [comment]
That's not falling in scope of what i envision Komga to do.
